### ISSUE TITLE
Installation problem of SPATA2 package

### ISSUE THREAD

**Issue by meiweijiucai** [theMILOlab/SPATA2#186]
The dependent packages c('BiocGenerics', 'DelayedArray', 'DelayedMatrixStats','limma', 'S4Vectors','SingleCellExperiment','SummarizedExperiment', 'batchelor', 'Matrix.utils', 'EBImage') were installed successfully. 

The confuns and SPATAData were also installed successfully. 

When I tried devtools::install_github(repo = "theMILOlab/SPATA2"), the following error occurred:

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/getMoleculeTypeList.Rd:20: missing link 'concept_molecular_modality'

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/getMolecules.Rd:113: missing link 'getSignatures'

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/getSasDf.Rd:51: missing link 'validUnitsofArea'

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/getSmrdResultsDf.Rd:10: missing link 'getSgsResults'

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/getSpatAnnArea.Rd:42: missing link 'validUnitsofArea'

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/getStsDf.Rd:39: missing link 'validUnitsofArea'

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/getTissueArea.Rd:23: missing link 'concept_area_measures'

Rd warning: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/loadSpataObject.Rd:10: missing link '`SPATA2''

error: C:/TempFiles/Rtmp8uMJBf/R.INSTALL2324a7302f/SPATA2/man/obsUnit.Rd:25: Bad \link option -- must be text
* removing 'C:/Program Files/R/R-4.1.2/library/SPATA2'

**Comment by kueckelj** [comment]
Hi, can you try again and see if the problem persists?

**Comment by willsonkenneth** [comment]
Yes, the problem still occurs. By using devtools::install_github(repo = "theMILOlab/SPATA2"), I got Error in utils::download.file(url, path, method = method, quiet = quiet,  : download from 'https://api.github.com/repos/theMILOlab/SPATA2/tarball/HEAD' failed

**Comment by RunyuXia** [comment]
Hi I'm encountering the same problem, just wondering if this has been solved?

**Comment by SvDenker** [comment]
Hi, I have the same problem and would love to know if you found a solution.

**Comment by simonmfr** [comment]
Thank you for reporting @willsonkenneth @RunyuXia @SvDenker. Using a clean environment and R4.5.1, I cannot reproduce that error. Did you follow the instructions on https://themilolab.github.io/SPATA2/articles/installation.html? In particular, did you make sure that all dependencies are installed and up to date?

**Comment by SvDenker** [comment]
I reinstalled all packages (there were problems with Matrix.utils), and yes I used exactly the code given in the instructions. Sstill using a clean environment in R4.5.0 gave me the same error. I found a solution forcing all dependent packages to update with pak and now SPATA2 is installed. Thanks.

**Comment by simonmfr** [comment]
Glad to hear. Please re-open if there are any related issues!
